modules/book.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ]  I would advise you to solve the Linter errors Manually and not disable it.

- Solve Linters errors or bugs, manually helps you to become an expert in debugging code. you can use [stackoverflow](https://stackoverflow.com/) website to search for how to solve bugs you are facing.